docs/content/en/concepts/apis-in-kcp.md

```diff
@@ -0,0 +1,376 @@
+
+### Define your APIExport
+
+An `APIExport` is the way an API provider makes one or more APIs (coming from `APIResourceSchemas`) available to
+workspaces.
+
+Here is an example `APIExport` called `example.kcp.dev` that exports 1 resource: `widgets`.
+
+```yaml
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: example.kcp.dev
+spec:
+  latestResourceSchemas:
+  - v220801.widgets.example.kcp.dev
+```
+
+At a minimum, you specify the names of the `APIResourceSchema`s you want to export in the `spec.latestResourceSchemas`
+field. The `APIResourceSchemas` must be in the same workspace as the `APIExport` (and therefore no workspace name or
+path is required here).
+
+You can optionally configure the following additional aspects of an `APIExport`:
+
+- its identity
+- what permissions consumers of your exported APIs are granted
+- API resources from other sources (built-in types and/or from other `APIExport`s) that your controllers need to access
+  for your service to function correctly
+
+We'll talk about each of these next.
+
+#### APIExport Identity
+
+Each API resource type is defined by an API group name and a resource name. Each API resource type can further be
+distinguished by an API version. For example, the `roles.rbac.authorization.k8s.io` resource that we frequently interact
+with is in the `rbac.authorization.k8s.io` API group and its resource name is `roles`.
+
+In a Kubernetes cluster, it is impossible to define the same API `<group>,<resource>,<version>` multiple times; i.e., it
+does not support competing definitions for `rbac.authorization.k8s.io,roles,v1` or any other API resource type.
+
+In kcp, however, it is possible for multiple API providers to each define the same API `<group>,<resource>,<version>`,
+_without interference_! This is where `APIExport` "identity" becomes critical.
+
+When you create an `APIExport` and you don't specify its identity (as is the case in the `example.kcp.dev` example
+above), kcp automatically generates one for you. The identity is always stored in a secret in the same workspace as
+the `APIExport.` By default, it is created in the `kcp-system` namespace in a secret whose name is the name of
+the `APIExport`.
+
+An `APIExport`'s identity is similar to a private key; you should never share it with anyone.
+
+The identity's **hash** is similar to a public key; it is not private and there are times when other consumers of your
+exported APIs need to know and reference it.
+
+Given 2 Workspaces, each with its own `APIExport` that exports `widgets.example.kcp.dev`, kcp uses the identity hash (in
+a mostly transparent manner) to ensure the correct instances associated with the appropriate `APIResourceSchema` are
+served to clients. See [Run Your Controller](#Run-Your-Controller) for more information.
+
+#### Permission Claims
+
+An API provider may need to access APIs that come from other APIExports, or that are built in to kcp. This is done by
```

Comment:
this seems to contradict the documentation in the code:

```
// PermissionClaim identifies an object by GR and identity hash.
// Its purpose is to determine the added permissions that a service provider may
// request and that a consumer may accept and allow the service provider access to.
```

Reply:
What part looks contradictory?

Reply:
Confusing more than contradict. I would rephase it, maybe something like this: "API provider may need to access resources in consumer workspaces. These resources might come from other APIExports, ...."

Does that make sense?

Reply:
Yes, thanks for the clarification

Reply:
Pushed an update - WDYT?

Reply:
much better. Thanks!